raddb/sites-available/default

```diff
@@ -1139,9 +1132,7 @@ send Access-Accept {
 	#  work, you will need to define the WiMAX NAI, usually
 	#  via:
 	#
-#	update request {
-#		&WiMAX-MN-NAI = "%{User-Name}"
-#	}
+#	&request.WiMAX-MN-NAI = "%{User-Name}"
```

Comment:
This will likely need to be updated to `&request.Vendor-Specific.WiMAX.......`.  But we can fix that in a later pass